frontend/packages/console-shared/src/components/TechPreviewBadge.tsx

```diff
@@ -2,7 +2,9 @@ import * as React from 'react';
 import { Label } from '@patternfly/react-core';
 
 const TechPreviewBadge: React.FC = () => (
-  <Label style={{ backgroundColor: 'var(--pf-global--warning-color--100)' }}>Tech Preview</Label>
+  <Label style={{ backgroundColor: '#D93F00', marginLeft: 'var(--pf-global--spacer--md)' }}>
```

Comment:
This cannot always have a margin left if it's a reusable component that can be used in different layouts.

Reply:
So, then have a prop like `withMargin` or `isInline` or something that controls the margin?